etcd-aws-cluster

```diff
@@ -16,6 +16,12 @@ server_port=${ETCD_SERVER_PORT:-2380}
 add_ok=201
 already_added=409
 delete_ok=204
+delete_gone=410
```

Comment:
Your are right. 410 return code is not mentioned in that doc (probably should make it clear.). 

The reason I included return http code "410" as an additional test was I ran into a condition during my frequent tear-down, build-up,  I sometimes got http 410 return code when trying to delete a machine that might have existed in autoscaling group but no longer part of etcd member. This condition is not covered in etcd remove member code.  I could not remember how exactly to re-produce that case now. it seems to okay to remove it, and just use 204 to test OK condition.  Thanks!